src/devtools/client/debugger/src/actions/breakpoints/modify.js

```diff
@@ -272,19 +254,14 @@ export function disableBreakpoint(cx, initialBreakpoint) {
   };
 }
 
-/**
- * Update the options of a breakpoint.
- *
- * @throws {Error} "not implemented"
- * @memberof actions/breakpoints
- * @static
- * @param {SourceLocation} location
- *        @see DebuggerController.Breakpoints.addBreakpoint
- * @param {Object} options
- *        Any options to set on the breakpoint
- */
 export function setBreakpointOptions(cx, location, options = {}) {
-  return async ({ dispatch, getState, client, sourceMaps }) => {
+  return async ({ dispatch, getState, client, parser }) => {
```

Comment:
We can keep this clean and remove the validation here if we disable the save button when there's an error in state.

Reply:
good call